lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -6008,67 +6016,40 @@ static SILFunction *createEmptyJVP(
                                 original->isDynamicallyReplaceable());
   jvp->setOwnershipEliminated();
   jvp->setDebugScope(new (module) SILDebugScope(original->getLocation(), jvp));
-  attr->setJVPName(jvpName);
 
   LLVM_DEBUG(llvm::dbgs() << "JVP type: " << jvp->getLoweredFunctionType()
              << "\n");
   return jvp;
 }
 
-/// Returns true on error.
 bool ADContext::processDifferentiableAttribute(
     SILFunction *original, SILDifferentiableAttr *attr,
     DifferentiationInvoker invoker) {
   auto &module = getModule();
-  // Try to look up JVP only if attribute specifies JVP name or if original
-  // function is an external declaration. If JVP function cannot be found,
-  // create an external JVP reference.
-  StringRef jvpName;
-  SILFunction *jvp = nullptr;
-  if (attr->hasJVP()) {
-    jvpName = attr->getJVPName();
-  } else if (original->isExternalDeclaration()) {
```

Comment:
Stdlib compilation and tests fail.
I wonder if it's related to the code here, which previously checked if `original->isExternalDeclaration()`.